packages/commercetools/composables/nuxt/README.MD

```diff
@@ -0,0 +1,53 @@
+# Example config
+```js
+export const config = {
+  disableMiddleware: false,
+  api: {
+    uri: 'https://api.commercetools.com/vsf-ct-dev/graphql',
```

Comment:
same - if it's a doc - don't use real access data. Put there something like "your-secret-key" etc.